README.md

```diff
@@ -1,3 +1,39 @@
 # Ophan GeoIP DB Refresher
 
+_Retrieves regular updates of the [MaxMind GeoIP Database](https://www.maxmind.com/en/geoip2-databases)
```

Comment:
thank you for adding this context 🙇‍♀️

Reply:
the regular updates are scheduled right? where does the scheduling live?

Reply:
That was just added with https://github.com/guardian/ophan-geoip-db-refresher/pull/3 !

Reply:
ohhh oops that's why I couldn't find it while poking around the code in intellij just now 😂